d8/d8batch/modules/d8feedsourcect/config/install/field.storage.node.field_feed_source_url.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - link
+    - node
+  enforced:
+    module:
+      - d8feedsourcect
+id: node.field_feed_source_url
+field_name: field_feed_source_url
+entity_type: node
+type: link
+settings: {  }
+module: link
+locked: false
+cardinality: 1
+translatable: true
```

Comment:
I don't really get what is translatable and what is not. To me it looks like the same stuff is both translatable and untranslatable. What's the story here?